extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -53,32 +53,30 @@ public class GelfConfig {
      * Negative throwable reference walk the exception chain from the root cause side: -1 will extract the root cause,
      * -2 the exception wrapping the root cause, ...
      */
-    @ConfigItem
-    public int stackTraceThrowableReference;
+    int stackTraceThrowableReference();
```

Comment:
My experience is that you will need a default value for this one and the boolean below (and maybe others). 

The defaults for primitives need to be explicit with config mapping.

Reply:
Should I the default int and boolean values?

Reply:
Yeah, for primitive ints, you need `@WithDefault("0")` if there is no defaults. For primitive booleans, you need `@WithDefault("false")` if there is no defaults.

Reply:
(Same for longs, shorts, ...)